client/client.go

```diff
@@ -33,9 +33,12 @@ type Config struct {
 
 func NewClient(c *Config) *Client {
 	// If endpoint field not specified, the info will be fetched from
-	// environment variable.
+	// environment variable.Otherwise using local host.
 	if c.Endpoint == "" {
-		c.Endpoint = os.Getenv("OPENSDS_ENDPOINT")
```

Comment:
Using the ENV to set the endpoint is ok, but one user want use the client  must set the ENV first everytime, I think it`s not very friendly. And user usually use the client in the local host, I think is if one user does not specfied the OPENSDS_ENDPOINT , using the the localhost as its default value maybe better.

Reply:
I know what you said, what I mean is maybe a better way is to pull an error if user doesn't specify the ENV. One use case is that for a new user, if he doesn't specify the endpoint ENV, you solution will assign 127.0.0.1:50040 to the variable, but if something goes wrong, user would be confused about the reported error since he didn't notice that he needs to configure the endpoint.
What do you think?

Reply:
Maybe we can print in log file to notice user, or get the endpoint infoamtion from config file.

Reply:
What about storing the default endpoint of both config and client in a unified file, and the client will be set to 127.0.0.1:50040 default?